src/Common/src/System/Runtime/InteropServices/StringBuffer.cs

```diff
@@ -42,6 +42,20 @@ public unsafe StringBuffer(string initialContents)
         }
 
         /// <summary>
+        /// Instantiate the buffer with a copy of the specified string.
+        /// </summary>
+        public unsafe StringBuffer(StringBuffer initialContents)
+            : base(0)
+        {
+            // We don't pass the count of bytes to the base constructor, appending will
+            // initialize to the correct size for the specified initial contents.
+            if (initialContents != null)
+            {
+                Append(initialContents);
+            }
```

Comment:
Right.  I get the desire for them if they're used a lot, but they don't seem to be.  In the StringBuilder case, the corresponding overloads also impact the initial capacity, and it doesn't seem to here.